src/appleseed-max-impl/appleseedrenderer/renderersettings.cpp

```diff
@@ -416,6 +481,74 @@ bool RendererSettings::save(ISave* isave) const
 
     isave->EndChunk();
 
+    //
+    // Stochastic Progressive Photon Mapping settings.
+    //
+
+    isave->BeginChunk(ChunkSettingsSPPM);
+
+        isave->BeginChunk(ChunkSettingsSPPMPhotonType);
+        success &= write<int>(isave, m_sppm_photon_type);
+        isave->EndChunk();
+
+        isave->BeginChunk(ChunkSettingsSPPMDirectLightingMode);
+        success &= write<int>(isave, m_sppm_direct_lighting_mode);
+        isave->EndChunk();
+
+        isave->BeginChunk(ChunkSettingsSPPMEnableCaustics);
+        success &= write<bool>(isave, m_sppm_enable_caustics);
+        isave->EndChunk();
+
+        isave->BeginChunk(ChunkSettingsSPPMEnableImageBasedLighting);
+        success &= write<bool>(isave, m_sppm_enable_image_based_lighting);
+        isave->EndChunk();
+
+        isave->BeginChunk(ChunkSettingsSPPMPhotonTracingMaxBounces);
+        success &= write<int>(isave, m_sppm_photon_tracing_max_bounces);
+        isave->EndChunk();
+
+        isave->BeginChunk(ChunkSettingsSPPMPhotonTracingDisableBounceLimit);
```

Comment:
Wouldn't it make more sense to call this chunk `ChunkSettingsSPPMPhotonTracingEnableBounceLimit`?

Reply:
yes, already changed it.